xml/MAIN-SBP-sumaforrhel.xml

```diff
@@ -987,7 +987,7 @@ suma:/ # unset URLGRABBER_DEBUG
             </listitem>
             <listitem>
                 <para>Best Practices - SUSE Manager 3.1: <link
-                        xlink:href="https://www.suse.com/documentation/suse-manager-3/book_suma_best_practices_3/data/mgr_expanded_support.html"
+                        xlink:href="https://documentation.suse.com/sbp/all/html/SBP-susemanager/index.html"
```

Comment:
This is a bad link, there is a SUSE Manager Best Practices guide in the SUSE Manager docs.

Reply:
Agree